api/envoy/config/core/v3/protocol.proto

```diff
@@ -386,6 +386,11 @@ message Http2ProtocolOptions {
   // Send HTTP/2 PING frames to verify that the connection is still healthy. If the remote peer
   // does not respond within the configured timeout, the connection will be aborted.
   KeepaliveSettings connection_keepalive = 15;
+
+  // `SETTINGS_MAX_HEADER_LIST_SIZE <https://tools.ietf.org/html/rfc7540>`_
```

Comment:
The link should be to the specific sub-section: https://tools.ietf.org/html/rfc7540#section-10.5.1

Reply:
Thanks for pointing it. I have updated.